src/style.css

```diff
@@ -1 +1,33 @@
-/* add styles here */
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 10px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 1.6rem;
+  height: 1.6rem;
+  margin-right: 4px;
```

Comment:
Be consistent with your margins. Add only top or bottom, but not both to avoid potential margin collapse. In this case, since you have a `margin-right` for `.stars__star`, ensure that you don't also add `margin-left` in any other rule that affects this element.